scrapers/DEU/rki.js

```diff
@@ -0,0 +1,42 @@
+import * as fetch from '../../lib/fetch.js';
+import * as parse from '../../lib/parse.js';
+
+const scraper = {
+  country: 'DEU',
+  timeseries: false,
+  url: 'https://services7.arcgis.com/mOBPykOjAyBO2ZKk/arcgis/rest/services/RKI_Landkreisdaten/FeatureServer/0/query?where=1%3D1&outFields=*&returnGeometry=false&outSR=4326&f=json',
+  type: 'json',
+  curators: [
+    {
+      name: 'Robert Koch Institute',
+      email: ''
```

Comment:
Why empty string here?

Reply:
The API is not exactly advertised by RKI, so I was not sure which mail to set